itou/approvals/management/commands/export_pe_api_rejections.py

```diff
@@ -59,6 +61,8 @@ def handle(self, **options):
             "prenom",
             "date_naissance",
             "siae_departement",
+            "siae_type",
```

Comment:
Peut-être plus facilement compréhensible car plus métier ? :shrug: 
```suggestion
            "siae_mesure",
```

Reply:
Oui, si ça peut éviter une éventuelle confusion, ça ne mange pas de pain.